Editor/MaterialSwitchEditorUtility.cs

```diff
@@ -14,6 +14,9 @@ namespace Unity.MaterialSwitch {
 /// Utility class to perform various functions on MaterialSwitch classes.
 /// </summary>
 public static class MaterialSwitchEditorUtility {
+    
+    private static Dictionary<Shader,MaterialPropertyNameMap> _nameRemaps;
```

Comment:
This is static, and not necessarily refreshed when we create a new MaterialPropertyNameMap asset

Reply:
@simonwittber I think you haven't addressed the above comment ?